discovery_engine_core/core/Cargo.toml

```diff
@@ -19,7 +19,7 @@ thiserror = "1.0.30"
 tokio = { version = "1.17.0", default-features = false, features = ["sync"] }
 url = { version = "2.2.2", features = ["serde"] }
 uuid = { version = "0.8.2", features = ["serde", "v4"] }
-xayn-ai = { git = "https://github.com/xaynetwork/xayn_ai", branch = "master" }
+xayn-ai = { git = "https://github.com/xaynetwork/xayn_ai", branch = "ty_2372_sort_when_no_coi" }
```

Comment:
This seems like why don't want to include it, right? Wouldn't we just wait until xaynetwork/xayn_ai#425 is merged?

Reply:
No we don't, was to have this already ready and change this to master when the other one is merged